cmd/podman/commit.go

```diff
@@ -80,7 +80,7 @@ func commitCmd(c *cli.Context) error {
 	case "oci":
 		mimeType = buildah.OCIv1ImageManifest
 		if c.IsSet("message") {
-			return errors.Errorf("messages cannot be added to the OCIv1 image format.")
+			return errors.Errorf("messages are only compatible with the docker image format (-f docker)")
```

Comment:
@rhatdan question, does the first docker here need to be capped?

Reply:
If you are specifying the command it can be lower case, all other cases it should be Upper.

Reply:
In this case we are talking about an option, so I would leave it lower case.